demo/admin/src/products/categories/ProductCategoriesPage.tsx

```diff
@@ -45,7 +65,65 @@ const ProductCategoriesPage: React.FC = () => {
                     {(selectedId) => (
                         <SaveBoundary>
                             <FormToolbar />
-                            <ProductCategoryForm id={selectedId} />
+                            <MainContent fullHeight>
+                                <RouterTabs>
+                                    <RouterTab
+                                        forceRender={true}
+                                        path=""
+                                        label={intl.formatMessage({ id: "products.editProductCategory.formTab", defaultMessage: "Product category" })}
+                                    >
+                                        <ProductCategoryForm id={selectedId} />
+                                    </RouterTab>
+                                    <RouterTab
+                                        forceRender={true}
+                                        path="/assigned-products"
+                                        label={intl.formatMessage({
+                                            id: "products.editProductCategory.assignedProducts",
+                                            defaultMessage: "Assigned Products",
+                                        })}
+                                    >
+                                        {/*<StackSwitch initialPage="table" title="TEST">*/}
+                                        {/*    <StackPage name="table">*/}
+                                        <Box sx={{ height: "100vh" }}>
+                                            <ProductsGrid filter={{ category: { equal: selectedId } }} />
+                                            {/* TODO change button behaviour and open edit-dialog, open discussion https://github.com/vivid-planet/comet/pull/2171 */}
+                                            <EditDialog
+                                                componentsProps={{
+                                                    dialog: { fullWidth: true, maxWidth: "xl" },
+                                                    dialogContent: {
+                                                        sx: { padding: 0, paddingTop: "0 !important" /* is connected to title-style */ },
+                                                    },
+                                                }}
+                                            >
+                                                <FinalForm<{ productIds: string[] }>
+                                                    mode="edit"
+                                                    onSubmit={async (values, form, event) => {
+                                                        await client.mutate<GQLSetProductCategoryMutation, GQLSetProductCategoryMutationVariables>({
+                                                            mutation: setProductCategoryMutation,
```

Comment:
I don't think it's best-practice to do a mutation in a page-component, I move this into a separate form-component